bin/population/pycbc_population_rates

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2017 Vaibhav Tiwari
+
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+Given an xml(.gz) table with CBC injections, this script separates them into:
+(1) (potentially) found injections
+(2) injections that we expect to miss
+The two sets are stored into two separate output files. 
+"""
+
+__author__ = "Vaibhav Tiwari"
+__email__ = "vaibhav.tiwari@ligo.org"
+__version__ = "0.0"
+__date__ = "31.10.2017"
+
+import logging
+import argparse
+import pycbc.version
+
+import os
+import h5py
+import pystan
+import numpy as np
+import scipy.stats as ss
+from pycbc.population import scale_injections as si
+from pycbc.population import rates_functions as rf
+import matplotlib.pyplot as plt
+
+import seaborn as sns
+import matplotlib as mpl
+
+# Parse command line
+parser = argparse.ArgumentParser(description='Estimate rates for flat in log and power-law mass distribution.')
+#parser.add_argument("--version", action="version",
+#                    version=pycbc.version.git_verbose_msg)
+#group = parser.add_mutually_exclusive_group(required=True)
+
+parser.add_argument('--sim-folder', dest='sim_folder', required=True, 
```

Comment:
Can you make the command line arguments take in the list of files it needs directly? Similar with the output produced. It should be explicit what files are being read and written.

Reply:
Alex this is not clear to me. Could you please explain ?

Reply:
Sure, I'd suggest the program take in the files directly instead of a folder. So on the command line you have

--inj-file inj1.hdf inj2.hdf ....
and possibly other options for other files you need. 

The point is that it's a bit dangerous to expect files to have some specific naming convention or to be laid out in a specific way. Also workflow managers generally require that these conditions never be assumed.